setup.py

```diff
@@ -131,8 +130,9 @@ def combine_files(*args):
     scripts=[
         'bin/tower-cli',
     ],
-    packages=find_packages(),
-
+    packages=[
+        'tower_cli',
+    ],
```

Comment:
Thanks for the review!

The problem is not that tests/ included in src distribution but bin distribution. For example, the result rpm by 'python setup.py bdist_rpm' (or archive by '... bdist') includes '.../site-packages/tests/...'. This is the problem.